common/src/common.py

```diff
@@ -73,4 +73,12 @@ def diff_and_save(data, fname, verbose=True):
   with open(fname, 'w') as f:
     json.dump(data, f)
 
-  return diff
\ No newline at end of file
+  return diff
+
+def normalize_state(data):
+  ''' Sort data in-place for consistent ordering '''
+  for datum in data:
+    for key in ['emails','phones','faxes']:
```

Comment:
To generalize this code for the future, can you do this for all fields where `isinstance(datum[key], list)`?

Reply:
Updated.